src/index.js

```diff
@@ -0,0 +1,178 @@
+        editAreaCbtn.classList.add('checkbtn');
+        const eCCancel = document.createElement('i');
+        eCCancel.classList.add('fa-solid');
+        eCCancel.classList.add('fa-xmark');
+        editAreaCbtn.append(eCCancel);
+        editContainer.append(editAreaCbtn);
+
+        editAreaCbtn.addEventListener('click', () => {
+          if (taskC.contains(editContainer) === true) {
+            editContainer.remove();
+          }
+          opt.classList.remove('vanish');
+          editTask.classList.add('vanish');
+          remove.classList.add('vanish');
+          window.location.reload();
+        });
+
+        const editArea = document.createElement('input');
+        editArea.value = toDoList[x].description;
+        editArea.classList.add('type-sec');
+        editContainer.append(editArea);
+        const editAreabtn = document.createElement('button');
+        editAreabtn.classList.add('checkbtn');
+        editContainer.append(editAreabtn);
+        const editEnter = document.createElement('i');
+        editEnter.classList.add('fa-sharp');
+        editEnter.classList.add('fa-solid');
+        editEnter.classList.add('fa-circle-check');
+        editAreabtn.append(editEnter);
+
+        editAreabtn.addEventListener('click', () => {
+          theTask.innerText = editArea.value;
+          toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+          toDoList[x].description = editArea.value;
+
+          if (taskC.contains(editContainer) === true) {
+            editContainer.remove();
+          }
+          opt.classList.remove('vanish');
+          editTask.classList.add('vanish');
+          remove.classList.add('vanish');
+          store();
+          window.location.reload();
+        });
+        taskC.append(editContainer);
+      });
+      remove.id = toDoList[x].id;
+      const removeTask = () => {
+        const target = remove.id;
+        toDoList.splice(target, 1);
+
+        let newId = 0;
+        if (toDoList.length > 0) {
+          toDoList.forEach((m) => {
+            m.id = newId;
+            newId += 1;
+          });
+        }
+      };
+
+      remove.addEventListener('click', () => {
+        toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+        removeTask();
+        store();
+
+        window.location.reload();
+      });
+    });
+  }
+};
+
+const enterTask = document.querySelector('.enter');
+enterTask.addEventListener('click', () => {
+  pushTask();
+  store();
+  toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+  window.location.reload();
+});
+
+const clearAll = document.querySelector('.clear-all');
+clearAll.addEventListener('click', () => {
+  toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+  toDoList = toDoList.filter((e) => e.completed === false);
+  let newId = 0;
+  if (toDoList.length > 0) {
+    toDoList.forEach((m) => {
+      m.id = newId;
+      newId += 1;
+    });
+  }
```

Comment:
After creating a class, all the functions in the file can be separated into different modules, then imported into the file and called in the methods of the class.